mailpoet/tests/integration/Automation/Engine/Control/StepHandlerTest.php

```diff
@@ -133,7 +133,7 @@ private function createWorkflow(): ?Workflow {
     $steps = [
       'root' => new Step('root', Step::TYPE_ROOT, 'root', [], [new NextStep('someone-subscribes')]),
       'someone-subscribes' => new Step('someone-subscribes', Step::TYPE_TRIGGER, $trigger->getKey(), [], [new NextStep('a')]),
-      'delay' => new Step('delay', Step::TYPE_ACTION, $delay->getKey(), [], []),
+      'delay' => new Step('delay', step::TYPE_ACTION, $delay->getKey(), [], []),
```

Comment:
Some typo? I guess PHP allows this...

Reply:
Addressed and squashed in [c61c118](https://github.com/mailpoet/mailpoet/pull/4452/commits/c61c118590447e5a1909053290c60a84002a51ee)